addrmgr/addrmanager.go

```diff
@@ -734,8 +740,25 @@ func (a *AddrManager) HostToNetAddress(host string, port uint16, services wire.S
 		if err != nil {
 			return nil, err
 		}
-		prefix := []byte{0xfd, 0x87, 0xd8, 0x7e, 0xeb, 0x43}
-		ip = net.IP(append(prefix, data...))
+
+		na = wire.NetAddressV2FromBytes(
+			time.Now(), services, data, port,
+		)
+	} else if len(host) == 62 && host[56:] == ".onion" {
```

Comment:
I think the new constants added in the `wire` package (when exported) can be used here instead of these raw numbers.